compatibility/history-proxy/js/tests/units/config/historyproxy.json

```diff
@@ -0,0 +1 @@
+{"logger":{"verbose":false,"console.level":"debug","file.enable":false,"file.level":"info","file.dir":"./temp/log/","file.name":"dojot.history-proxy-%DATE%.log","file.max":"7d","file.size":"10m"},"server":{"host":"0.0.0.0","port":18000,"shutdown.delay":10000,"shutdown.gracefultimeoutms":60000,"shutdown.handlertimeoutms":5000,"healthcheck.port":9000,"history.hostname":"history","history.port":8000,"history.protocol":"http:","retriever.hostname":"influxdb-retriever","retriever.port":3000,"retriever.protocol":"http:"}}
```

Comment:
do not commit this file

Reply:
Thank you! The Jest mocking schema was added for Config Manager methods.